===================================================================
RCS file: /cvsroot/kmess/kmess/kmess/notification/balloonbase.cpp,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- kmess/kmess/kmess/notification/balloonbase.cpp	2005/07/13 14:49:44	1.6
+++ kmess/kmess/kmess/notification/balloonbase.cpp	2006/02/15 20:55:25	1.7
@@ -48,7 +48,7 @@
   messageLabel_ = new KURLLabel( this, "messageLabel" );
   messageLabel_->setText( "" );
   messageLabel_->setAlignment( int( QLabel::AlignCenter | QLabel::AlignVCenter ) );
-  messageLabel_->setSizePolicy( QSizePolicy::Minimum, QSizePolicy::Minimum );
+  messageLabel_->setSizePolicy( QSizePolicy(QSizePolicy::Minimum, QSizePolicy::Minimum) );
   messageLabel_->setUnderline( false );
   balloonLayout_->addMultiCellWidget( messageLabel_, 0, 1, 2, 2 );
 
